www/include/options/media/images/formImg.php

```diff
@@ -216,7 +220,7 @@
 }
 $action = $form->getSubmitValue("action");
 
-if ($valid) {
+if (is_array($valid)) {
```

Comment:
strange validation. I think methods need to return "false" if update or upload doesn't work